js/widgets/forms/rangeslider.js

```diff
@@ -40,17 +40,14 @@
 		options: {
 			theme: null,
 			trackTheme: null,
-			corners: true,
-			mini: false,
-			highlight: true
+			classes: {}
 		},
 
 		_create: function() {
-			var $el = this.element,
-			elClass = this.options.mini ? "ui-rangeslider ui-mini" : "ui-rangeslider",
-			_inputFirst = $el.find( "input" ).first(),
-			_inputLast = $el.find( "input" ).last(),
-			_label = $el.find( "label" ).first(),
+			var rangeslider = this.element,
```

Comment:
lets drop the extra `var` here `rangeslider` is only 1 char shorter then `this.element` and gzip likes repeated strings :-)